MobileWallet/TariLib/TariLib.swift

```diff
@@ -122,7 +122,8 @@ class TariLib {
                databaseName: TariLib.databaseName,
                publicAddress: publicAddress,
                discoveryTimeoutSec: TariSettings.shared.discoveryTimeoutSec,
-               safMessageDurationSec: TariSettings.shared.safMessageDurationSec
+               safMessageDurationSec: TariSettings.shared.safMessageDurationSec,
+               networkName: "weatherwax"
```

Comment:
We keep network names in `TariNetwork` enum. Please consider this suggestion:
```suggestion
               networkName: TariNetwork.weatherwax.rawValue
```

Reply:
While I could do this, this would need to change further when the UI for network selection is done. So would end up being something like `TariSettings.shared.currentNetwork`.

Thoughts?

Reply:
Ok, so if we want to be able to switch between networks then your solution will fit better here. However, this still will be a static and imperative way to do that. 

But, let's don't try to fix future problems. I'll handle it properly when I will be working on implementing that selector in the future.

Reply:
So, please replace hardcoded string with `TariSettings.shared.currentNetwork.value`